src/Components/List/CategoryList.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import categoriesData from "./../../data/mocks/es-mx/product-categories.json";
+
+export default function CategoryList(props) {
+  const [seeMore, setSeeMore] = useState(false);
+  let lastItem = props.itemsPerRow;
```

Comment:
most of the time the props are [destructured](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to reduce the noise when there is multiple `props.`

```suggestion
export default function CategoryList({ itemsPerRow: lastItem, start }) {
  const [seeMore, setSeeMore] = useState(false);
```